src/main/notifications/index.js

```diff
@@ -24,9 +26,20 @@ export function displayMention(title, body, channel, teamId, silent, webcontents
         silent,
         data,
     };
-    const mention = new Mention(options);
+
+    const mention = new Mention(options, channel, teamId);
 
     mention.on('show', () => {
+        // On Windows, manually dismiss notifications from the same channel and only show the latest one
+        if (process.platform === 'win32') {
+            for (const notification of currentNotifications.values()) {
+                if (notification.channel.id === mention.channel.id && notification.teamId === mention.teamId) {
+                    notification.close();
+                    currentNotifications.delete(notification);
```

Comment:
if the notification was closed by user action, will a second close on it fail somehow?

Reply:
@jgilliam17 on this test case ^